guides/micronaut-cloud-trace-base/create-app.adoc

```diff
@@ -0,0 +1,75 @@
+common:completesolution.adoc[]
+
+common:create-app-features.adoc[]
+
+=== OpenTelemetry
+
+The Micronaut framework uses https://opentelemetry.io/[OpenTelemetry] to generate and export tracing data.
+
+OpenTelemetry provides two annotations one to create a span and another to include additional information to the span.
+
+@WithSpan:: Used on methods to creates a new span using a given name or defaults to the method name.
+
+@SpanAttribute:: Used on method parameter to include the value to the span with a given name or defaults to the parameter name.
+
+The `@WithSpan` and `@SpanAttribute` can be used on non-private methods.
+
+If the annotations are not enough, or you want to add tracing to a private method, Micronaut’s tracing integration registers a `io.opentelemetry.api.trace.Tracer` bean which exposes the OpenTelemetry API and can be dependency-injected as needed.
+
+[NOTE]
+====
+The `io.micronaut.tracing.annotation` are available if you prefer or have an existing Micronaut Application already using them.
+
+- `@NewSpan` : Identical to `@WithSpan`, it is used on methods to creates a new span using a given name or defaults to the method name.
+- `@ContinueSpan` : Used on methods to continue the existing span. Primarily used in conjunction with `@SpanTag`.
+- `@SpanTag` : Similar to `@SpanAttribute`, it is used on method parameter to include the value to the span with a given name or defaults to the parameter name. To use the `@SpanTag` on a method argument, the method must be annotated with either `@NewSpan` or `@ContinueSpan`.
+====
+
+==== OpenTelemetry Trace Exporter
+If you did not use Micronaut CLI or Launch to create your application, add the OpenTelemetry exporter to your `application.yml`.
```

Comment:
This sentence makes sense if we also add list of dependencies that should end user have to add.

Reply:
Removed that section. Since OCI requires extra otel configurations.